### PR TITLE
DYN-6515 OpenXML ExportXML Bug Fix

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Fixing some bugs found when using the node Data.OpenXMLExportExcel.
When using the Data.OpenXMLExportExcel node was generating a error saying "Object reference not set to an instance of an object", then in this fix I'm adding code for preventing that error and also fixed the method call to sheetData.InsertBefore due that the arguments were inverted and were producing a node warning message when the spreadsheet used in Data.OpenXMLExportExcel contains empty space in the middle of the populated rows.
### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

Fixing some bugs found when using the node Data.OpenXMLExportExcel.

### Reviewers

@reddyashish @zeusongit 

### FYIs

@jnealb

### CODE REVIEW

**Comment by RobertGlobant20** [comment]
GIF showing the expected behavior using the files attached in the Jira task DYN-8615 (files added when a bug was found during testing).
devenv_oKGH7SDe3C

**Review by zeusongit — APPROVED** [review]

File: src/Libraries/DSOfficeUtilities/OpenXmlHelper.cs [hunk 1]
```diff
@@ -608,6 +612,14 @@ private static void SetCellValue(object value, Cell cell, SharedStringTable shar
                 {
                     // The string is not in the table, so we add it
                     sharedStringTable.AppendChild(new SharedStringItem(new Text((string)value)));
+
+                    //Means that the current (row, column) is probably empty so we should start from 0
+                    if (sharedStringTable.Count == null)
+                    {
+                        sharedStringTable.Count = 0;
+                        sharedStringTable.UniqueCount = 0;
```
Comment (aparajit-pratap) [inline src/Libraries/DSOfficeUtilities/OpenXmlHelper.cs:620]: I don't get this logic. Are `Count` and `UniqueCount` the row and column numbers, respectively?

Comment (RobertGlobant20) [reply src/Libraries/DSOfficeUtilities/OpenXmlHelper.cs:620]: The Book2.xlsx (attached in the Jira task) has the first row as empty, so when reading worksheet with OpenXML `document.WorkbookPart.SharedStringTablePart.SharedStringTable.Count is null also ...UniqueCount`
then the code is trying to access both Count and UniqueCount that's why we get the `**Object reference not set to an instance of an object**`. message.

Microsoft documentation about SharedStringTable says:
**_SharedStringTable - An instance of this part type contains one occurrence of each unique string that occurs on all worksheets in a workbook.
Count - An integer representing the total count of strings in the workbook
UniqueCount - An integer representing the total count of unique strings in the Shared String Table
_**
Then I guess due that we have an empty row at the beginning of the spreadsheet when reading Book2.xlsx with OpenXML SharedStringTable is empty and SharedStringTable.Count and SharedStringTable.UniqueCount is null.

I think I can debug more and get more info about why SharedStringTable is empty but is more related in how the xlsx file was created or edited (tested with a spreadsheet without empty rows and SharedStringTable has items).

image

**Comment by zeusongit** [comment]
Merging, tests to follow in a separate PR after discussion with @RobertGlobant20
